overlord/fdestate/backend/seal.go

```diff
@@ -210,13 +212,13 @@ func sealKeyForBootChainsBackend(method device.SealingMethod, key, saveKey secbo
 
 	// TODO:FDEM: refactor sealing functions to take a struct instead of so many
 	// parameters
-	primaryKey, err = sealRunObjectKeys(key, pbc, primaryKey, volumesAuth, params.RoleToBlName, handle, params.UseTokens)
+	primaryKey, err = sealRunObjectKeys(key, pbc, primaryKey, volumesAuth, params.RoleToBlName, handle, params.UseTokens, "run+recover")
```

Comment:
Would it not make sense to either strongly type the possible keyroles or atleast make them `const` defines somewhere?

Reply:
Yes. It is already lots of TODOs about it.